From d841c19ce7bf472b7e9a04cd3e4179c79f95c32c Mon Sep 17 00:00:00 2001
From: Thomas Senyk <thomas.senyk@pelagicore.com>
Date: Tue, 15 Jan 2013 14:54:45 +0100
Subject: [PATCH] SourceProxy.qml: removing binding 'output: input'

The binding is redundant as onInputChanged will
always trigger setting output
Also the binding does generate binding-loop errors

Change-Id: Iaa1df71a633f4bbd0554d81bb807db59a02974ee
Reviewed-by: Gunnar Sletta <gunnar.sletta@digia.com>
---
 src/effects/private/SourceProxy.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/effects/private/SourceProxy.qml b/src/effects/private/SourceProxy.qml
index ef60a3e..2561d70 100644
--- a/src/effects/private/SourceProxy.qml
+++ b/src/effects/private/SourceProxy.qml
@@ -43,7 +43,7 @@ import QtQuick 2.0
 Item {
     id: rootItem
     property variant input
-    property variant output: input
+    property variant output
     property variant sourceRect
     visible: false
 
-- 
1.8.0

